docs/pages/management/operations/ca-rotation.mdx

```diff
@@ -25,6 +25,13 @@ configured in group policy.
 [Read more about exporting the Teleport CA](../../desktop-access/manual-setup.mdx#export-the-teleport-ca)
 </Admonition>
 
+<Admonition type="warning" title="Certificate Authority Types">
```

Comment:
Since the intro to the "Certificate Authority rotation" H2 now has more text inside an `Admonition` box than outside, this component loses its ability to signal importance.

What would you think of removing the `Admonition` tags from this and the previous `Admonition`, and treating all of the paragraphs before the "Rotation phases" heading as regular body text?

Since we mention this information below, we could also remove this `Admonition` entirely here.

Reply:
This is a change to a default behavior that is being deprecated and requires user changes so I feel like it should stay here. This should only be for a single release and then we can remove it entirely in v13.